CHANGES.rst

```diff
@@ -41,6 +41,8 @@ Cubeviz
 Imviz
 ^^^^^
 
+- ASDF parser for JWST images now uses ``stdatamodels``. [#2052]
```

Comment:
Maybe we should use more specific language to prevent confusion with #1822.
```suggestion
- ASDF-in-FITS parser for JWST images now uses ``stdatamodels``. [#2052]
```

Reply:
Good point. Done. Thanks!